hack/tools/release/docker-image-verify.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
Let's keep this directly in hack/ for now like all our other scripts

Reply:
Updated.